dbt/adapters/databricks/impl.py

```diff
@@ -59,6 +59,7 @@ class DatabricksConfig(AdapterConfig):
     options: Optional[Dict[str, str]] = None
     merge_update_columns: Optional[str] = None
     tblproperties: Optional[Dict[str, str]] = None
+    zorder: Optional[Union[List[str], str]] = None
```

Comment:
Is this configuration per-relation? Or per project?

Reply:
you can define dbt configs on a per project, per folder, or per model basis